content/en/blog/2023/end-user-discussions-04.md

```diff
@@ -0,0 +1,111 @@
+organization?
+
+**A:** In a big organization, the first step would be to put out the current
+pain points of observability to leadership. There are benefits of having an open
+source standard for observability. If there is no standard in place, it gets
+very difficult to communicate across different teams. If you use OpenTelemetry,
+you do not have to depend on any vendor agents, and you have the flexibility to
+send data to multiple backends.
+
+### How to optimize observability data at scale?
+
+**Q:** In a big organization, observability data can be in the range of TBs per
+day, which comes with associated costs. But there is always a feeling that 80%
+of captured data is unusable. None of the vendors help you understand what data
+is accessed and how to bring that visibility to the engineering teams sending
+the data.
+
+**A:** One of the ways to optimize observability at scale is sampling. There are
+a number of options for you to reduce the data volumes at the SDKs level and the
+collector level.
+
+Also, there is active work going on the OpenTelemetry Collector side to handle
+data at scale more efficiently. For example, there is work going around using
+Apache Arrow for serialization to optimize network costs.
+
+One of the other ways is to look at retention settings for the collected
+telemetry data. There is no need to retain data that won't be used later.
+
+## Other Important discussion points
+
+#### Maturity model for OpenTelemetry
+
+**Q:** Is there some literature available around understanding steps to reach a
+certain maturity level in adopting OTel in your organization? For example, I
+should be able to go to a team and tell them to start with X, and then do Y to
+move ahead. In a big enterprise, you have to provide something for people to
+understand the maturity of OpenTelemetry.
+
+**A:** OpenTelemetry metrics is a good place to start for infrastructure
+monitoring. For teams adopting OpenTelemetry, a good idea is to start with
+minimal changes. For example, teams can start with languages that have
+auto-instrumentation support. Seeing value from small changes can build more
+confidence in the team to go deeper into OpenTelemetry adoption.
+
+There are also several OpenTelemetry receivers
+[available](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver).
+Using these receivers, you can start sending telemetry data from different
+components of your application.
+
+#### Roadmap for auto instrumentation in OpenTelemetry
+
+**Q:** Is there any roadmap for auto instrumentation in OpenTelemetry?
+
+**A:** Auto instrumentation is not a priority for OpenTelemetry right now. It is
+also something that is driven by the community. If you're interested in the
+auto-instrumentation of a language or want to improve on existing areas, you can
+take it up. On a side note, the first version of Golang auto instrumentation
+just got released.
```

Comment:
```suggestion
```

the otel roadmap specifically calls out that continued investment in OpenTelemetry Artifacts is a top priority and lists auto instrumentations as one of them (https://opentelemetry.io/community/roadmap/#p0-continued-investment-in-opentelemetry-artifacts). 

Indeed there is no explicit roadmap, so my suggestion is dropping this whole Question

Reply:
Seconded.

Reply:
Thanks for pointing this out @svrnm